frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -165,9 +165,14 @@ class Sidebar extends Component<Props> {
             <div className={['sidebarContainer', hidden ? 'forceHide' : ''].join(' ')}>
                 <div className={['sidebarOverlay', activeSidebar ? 'active' : ''].join(' ')} onClick={toggleSidebar}></div>
                 <nav className={['sidebar', activeSidebar ? 'forceShow' : '', shown ? 'withGuide' : ''].join(' ')}>
-                    <div className="sidebarLogoContainer">
-                        <AppLogoInverted className="sidebarLogo" />
-                    </div>
+                    <Link
```

Comment:
is Link the right component? (don't know what it does). could be `<a ...>` as well?

@originalblend337

Reply:
Link is fine

Reply:
So should I use `Link` or `<a>` or something else?

Reply:
Pleaes keep using the Link component from preact-router.